rest/route/build.go

```diff
@@ -53,9 +53,18 @@ func (b *buildGetHandler) Run(ctx context.Context) gimlet.Responder {
 	for _, t := range foundBuild.Tasks {
 		taskIDs = append(taskIDs, t.Id)
 	}
-	tasks, err := data.FindTasksByIds(taskIDs)
-	if err != nil {
-		return gimlet.MakeJSONInternalErrorResponder(errors.Wrap(err, "Database error"))
+	var tasks []task.Task
+	if len(taskIDs) > 0 {
+		tasks, err = task.Find(task.ByIds(taskIDs))
+		if err != nil {
+			return gimlet.MakeJSONInternalErrorResponder(errors.Wrap(err, "Database error"))
+		}
+		if len(tasks) == 0 {
+			return gimlet.MakeJSONInternalErrorResponder(errors.Wrap(gimlet.ErrorResponse{
```

Comment:
there's no reason to wrap it this way. We can just return the gimlet.ErrorResponse wrapped directly by MakeJSONErrorResponder (All "Internal" does is add the http.StatusInternalError status code, which we'd be overwriting anyway). 

However, since _we_ determine the task Ids and they aren't passed in, I think a internal server error is correct since it's our fault, so we can just return `gimlet.MakeJSONInternalErrorResponder(errors.New("no tasks found"))` or something